mercure.rst

```diff
@@ -211,7 +210,7 @@ as patterns:
 .. code-block:: javascript
 
     // URL is a built-in JavaScript class to manipulate URLs
-    const url = new URL('http://localhost:3000/.well-known/mercure');
+    const url = new URL('/.well-known/mercure');
```

Comment:
Is this working? :thinking: 

From the [`URL` documentation on MDN](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#parameters) :

> If `url` is a relative URL, `base` is required, and will be used as the base URL. If `url` is an absolute URL, a given base will be ignored.

Output from Chromium's console : 
```
> const url = new URL('/.well-known/mercure');
VM89:1 Uncaught TypeError: Failed to construct 'URL': Invalid URL
    at <anonymous>:1:14
```

Reply:
Good catch, window.origin must be passed as 2nd parameter, I'll fix this.